src/java/org/apache/cassandra/transport/PreV5Handlers.java

```diff
@@ -327,6 +330,13 @@ public void exceptionCaught(final ChannelHandlerContext ctx, Throwable cause)
                 logger.debug("Excluding client exception for {}; address contained in client_error_reporting_exclusions", ctx.channel().remoteAddress(), cause);
                 return;
             }
+
+            if (Throwables.anyCauseMatches(cause, t -> t instanceof SSLException))
+            {
+                logger.warn("SSLException in client networking with peer {} {}", ctx.channel().remoteAddress(), cause.getMessage());
```

Comment:
Agree with @Maxwell-Guo . Ping me in the ticket once addressed. Thanks

Reply:
That makes sense to me, although moving to `logClientNetworkingExceptions()` looks like we lose the `ctx` variable that provides the host we tried to connect to. Should we still move to `logClientNetworkingExceptions`?

I also added a call to `ClientMetrics.instance.markProtocolException();` since it isn't too much trouble.

Reply:
@jameshu15869  just put `ctx.channel().remoteAddress()` into `logClientNetworkingExceptions` as the second parameter of that method and move that `if` there and log it with NoSpamLogger.

Reply:
yes, it feels like a different metric, not protocol exception one

Reply:
Makes sense - I added a `SslHandshakeException` metric following the `ProtocolException` model.

Reply:
This is mostly a cosmetic problem so I don't think metrics or a test are strictly necessary.

Reply:
A new metric like this is pretty harmless, why not to add it? As Brandon said, a test is not strictly necessary. I am on the same page there. I think the PR is good as is. As I said on the ticket, I will try to build this in a foreseeable future.